trie/statistics/trieSyncStatistics.go

```diff
@@ -110,6 +131,22 @@ func (tss *trieSyncStatistics) NumTries() int {
 	return len(tss.missingMap)
 }
 
+// ProcessorTime will return the cumulative processor time used in tries synchronization (sum of all go routines used in trie sync processes)
+func (tss *trieSyncStatistics) ProcessorTime() time.Duration {
```

Comment:
might rename `ProcessorTime` / `AddProcessorTime` to `ProcessingTime` / `AddProcessingTime`

Reply:
or `CumulatedTime` / `AddToCumulatedTime`

Reply:
done